model/src/main/java/com/basistech/rosette/dm/RelationshipMention.java

```diff
@@ -192,8 +194,8 @@ public Double getSalience() {
                 .add("temporals", temporals)
                 .add("source", source)
                 .add("confidence", confidence)
-                .add("salience", salience);
-
+                .add("salience", salience)
+                .add("modality", modality == null ? "{}" : Iterables.toString(modality));
```

Comment:
I don't see the special check for null on any of the other toStringHelper methods in the adm.  Why here?  It the underlying code can't take null, we have lots of bugs.  If it can, this is merely inconsistent.